allennlp/training/callbacks/log_writer.py

```diff
@@ -227,7 +227,7 @@ def log_batch(
 
             # Now collect per-batch metrics to log.
             metrics_to_log: Dict[str, float] = {}
-            for key in ("batch_loss", "batch_reg_loss"):
+            for key in ["batch_loss", "batch_reg_loss"]:
```

Comment:
Tuples have a smaller overhead which is why I tend to do this sort of thing that way. But if we really cared about things like "overhead" we wouldn't be using Python 😜  So 🤷‍♂️

Reply:
I didn't have to change this. When I read code I tend to make little tweaks like this to stay focused. In principle what we want there is an ordered set. In practice it doesn't matter.